### ISSUE TITLE
useRelativePaths = false causes local modules to be grouped with packages

### ISSUE THREAD

**Issue by rhettlivingston** [Galooshi/import-js#305]
With useRelativePaths = true, import-js creates the following imports in one of my modules
```js
import { DDPRateLimiter } from 'meteor/ddp-rate-limiter';
import { Meteor } from 'meteor/meteor';

import { SimpleSchema } from 'meteor/aldeed:simple-schema';
import { ValidatedMethod } from 'meteor/mdg:validated-method';
import { _ } from 'meteor/underscore';

import { Lists } from '../lists/lists';
import { Todos } from './todos';
```

When I set useRelativePaths = false, the two local modules moved into the packages section
```js
import { DDPRateLimiter } from 'meteor/ddp-rate-limiter';
import { Meteor } from 'meteor/meteor';

import { Lists } from 'imports/api/lists/lists';
import { SimpleSchema } from 'meteor/aldeed:simple-schema';
import { Todos } from 'imports/api/todos/todos';
import { ValidatedMethod } from 'meteor/mdg:validated-method';
import { _ } from 'meteor/underscore';
```

Note that I am working on the packageDependencies-is-configurable branch though I believe this is independent of that. 

Do I correctly remember that there is a point in the code that looks at the leading "." in the path to determine whether it is dealing with a package?

**Comment by trotzig** [comment]
I think the bug here is that they are never treated as package dependencies. Regular relative imports end up in a different group than regular absolute imports. I think that's what's causing the first example to look "correct". 

There's a check in lib/ImportStatements.js that you might have to adjust for meteor: https://github.com/Galooshi/import-js/blob/7d8dfbff30fe5bb3809fd1c8b78a6dce1705bc83/lib/ImportStatements.js#L69

**Comment by trotzig** [comment]
Sorry, didn't mean to close.

**Comment by rhettlivingston** [comment]
Yes, you're correct. I misinterpreted what was happening.

**Comment by rhettlivingston** [comment]
And now I was too hasty. :smile: 

You were correct in the reason it was happening at that time. But now that I've got the packageDependencies properly specified, I still have the problem for the right reason.

You were also correct in that the aforementioned line in lib/ImportStatements.js needs an adjustment for Meteor.

I think the best fix is to replace 

```js
  // If there is a slash in the path, remove that and everything after it.
  // This is so that imports for modules inside package dependencies end up
  // in the right group (PATH_TYPE_PACKAGE).
  const path = importStatement.path.replace(/^(.*?)\/.*$/, '$1');
  if (packageDependencies.indexOf(path) !== -1) {
    return PATH_TYPE_PACKAGE;
  }
```
with this
```js
  // Match if any of the packageDependencies exactly match path or match the
  // start of the path up to a path divider. This is so that imports for
  // modules inside package dependencies end up in the right group
  // (PATH_TYPE_PACKAGE).
  if (packageDependencies.some((pkg: string): boolean =>
        importStatement.path === pkg ||
        importStatement.path.startsWith(`${pkg}/`))) {
    return PATH_TYPE_PACKAGE;
  }
```

As compared to changing the regex to pull in 'meteor/' prefixes, this fix has the advantage of not being Meteor specific.

I'm checking the tests for impact now.

**Comment by trotzig** [comment]
I believe this was fixed by fd7b809fc1.
